spec/api/comment_thread_spec.rb

```diff
@@ -477,6 +477,18 @@ def test_unicode_data(text)
         check_thread_result_json(nil, thread, parsed)
       end
 
+      context 'when requesting the thread for informational purposes' do
+        subject do
+          get "/api/v1/threads/#{thread.id}", with_responses: false # we're asking for no responses here.
```

Comment:
I'm debating whether a balanced test with `with_responses: true` and a check for `'children'` would make sure that at least one of these 2 tests would fail if anything ever changed with the api.  Especially if `'children'` were a constant used in both tests.